### DISCUSSION TITLE
Exempt players from inspection list.

### DISCUSSION THREAD

**Discussion by Morphiee** [BG-Software-LLC/WildInspect#25] [Pending Feature Requests]
Hello! There was a very vague request from 2021 so I figured I would add my own with a little more information.

Currently the config has an option to exempt operators or hide them from being in inspections. (at least I assume that's how it is suppose to work) Now, not all of my staff are operators. Having a way to exempt other members of staff say via a permission? (wildinspect.exempt) That way we can set that permissions to the staff roles and easily remove my staff from being in inspection lists.

**Comment by OmerBenGera** [comment]
That's not really possible as the plugin cannot access player permissions while the player is offline.

**Reply by Morphiee** [reply]
Yeah that's the issue I was running into.. The only way I could possibly think of is using vault. I want to say vault can pull offline player permissions. Though I know for sure luck perms requires it to be async. Caching players onEnable would be the only way potentially but I also am speaking hypothetically here as I don't know how feasible that is. 

But if somehow those players can be stored and updated when joining? That list can then be checked during inspections. Again though, that's a big if.

EDIT: Just because I had a random thought. Alternatively, I wonder if an “ignored players” list is worth it? Yeah users would have to manually update it but at least its an easy addition.

**Reply by OmerBenGera** [reply]
> Yeah that's the issue I was running into.. The only way I could possibly think of is using vault. I want to say vault can pull offline player permissions. Though I know for sure luck perms requires it to be async. Caching players onEnable would be the only way potentially but I also am speaking hypothetically here as I don't know how feasible that is.
> 
> But if somehow those players can be stored and updated when joining? That list can then be checked during inspections. Again though, that's a big if.
> 
> EDIT: Just because I had a random thought. Alternatively, I wonder if an “ignored players” list is worth it? Yeah users would have to manually update it but at least its an easy addition.

An "ignored players" list sounds fine for me.
A PR would be more than welcomed with these changes.
